docs/api/modules.rst

```diff
@@ -0,0 +1,7 @@
+microraiden
+===========
+
+.. toctree::
+   :maxdepth: 4
+
+   microraiden
```

Comment:
Is there a current issue with the build or am I missing something? (there is no reference to the api in the docs and are not built correctly)

Reply:
Fixed in #436, that was supposed to be merged before any docs merge to master

Reply:
So #436 is not ready yet I assume. Ok.

Reply:
@ezdac Could you explain why we have the `docs/api/..` folder?
Btw it compiles fine for me. @loredanacirstea can you access the files by manually entering the URL?

Reply:
yes, I can access them, but I only saw headlines with no other content.